### PR TITLE
[`recordrtc`] Fixed some properties of `RecordRTC`

### PR DESCRIPTION
**Pull request by hrueger**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/muaz-khan/RecordRTC/blob/master/RecordRTC.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## recordrtc ([<kbd>unpkg</kbd>](https://unpkg.com/browse/recordrtc@latest/))
was missing the following properties:
1. getFromDisk
1. writeToDisk
1. MRecordRTC
1. getTracks
1. getSeekableBlob

<details>
<summary>as well as these 12 other properties...</summary>
<p>isElectron, Storage, MediaStreamRecorder, StereoAudioRecorder, CanvasRecorder, WhammyRecorder, Whammy, DiskStorage, GifRecorder, MultiStreamRecorder, RecordRTCPromisesHandler, WebAssemblyRecorder</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 44c6b36657a4869159a4ff621e0ed98d9fabb9c5
</p>

**Comment by imnasnainaec** [comment]
@hrueger `inferface Options` is missing `recorderType?: MediaStreamRecorder | StereoAudioRecorder | WebAssemblyRecorder | CanvasRecorder | GifRecorder | WhammyRecorder`

**Comment by hrueger** [comment]
Yeah, I'll fix that tomorrow.

File: types/recordrtc/index.d.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ declare namespace RecordRTC {
         frameInterval?: number;
 
         /** if you are recording multiple streams into single file, this helps you see what is being recorded */
-        previewStream?(cb: (stream: MediaStream) => void): void;
+        previewStream?: (stream: MediaStream) => void;
```
Comment (k-yle) [inline types/recordrtc/index.d.ts:57]: oops, these three (`previewStream`, `onTimeStamp`, `ondataavailable`) were wrong initially. Not sure why I thought they were callbacks. This new version is correct.

File: types/recordrtc/recordrtc-tests.ts [hunk 2]
```diff
@@ -9,6 +9,15 @@ navigator.getUserMedia(
             type: 'video',
             disableLogs: true,
             bufferSize: 2048,
+            ondataavailable: (blob: Blob) =>
```
Comment (k-yle) [inline types/recordrtc/recordrtc-tests.ts:12]: The CI tests are failing because you're missing a `{` at the end of this line

**Review by k-yle — CHANGES_REQUESTED** [review]
Looks good so far! See my comment on the test file. Hit the 🔄 button if you make any more changes and I can re-review

File: types/recordrtc/recordrtc-tests.ts (hunk 2 above)
Comment (hrueger) [reply types/recordrtc/recordrtc-tests.ts:12]: Oops, I did that on my phone yesterday where the GitHub UI has neither syntax highlighting nor auto completion. Sorry for that!

**Review by k-yle — APPROVED** [review]
Some more unit tests would be ideal, but they can be added later by whoever adds detailed definitions for the recorderType classes. 

Looks good to me, thank you.

**Comment by hrueger** [comment]
Ready to merge

**Comment by imnasnainaec** [comment]
Thanks for doing this! The `mimeType` options are still incomplete, so I'll probably make a new pr for that (if nobody beats me to it).

**Pull request merged**